apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/TimedBufferingConsumer.cs

```diff
@@ -15,19 +15,18 @@
 using Google.Api.Gax;
 using System.Collections.Generic;
 using System;
+using System.Threading.Tasks;
+using System.Threading;
 
 namespace Google.Cloud.Diagnostics.Common
 {
     /// <summary>
     /// A <see cref="IFlushableConsumer{T}"/> that will automatically flush after
     /// a given span of time.
     /// </summary>
-    internal class TimedBufferingConsumer<T> : IFlushableConsumer<T>
+    internal class TimedBufferingConsumer<T> : FlushableConsumerBase<T>
```

Comment:
Is the description actually accurate here? It only flushes if Receive (or flush) is called again after that time, There's no background flush going on, is there?